server/pages/index.jade

```diff
@@ -115,6 +115,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script(src='/calypso/polyfill.js')
```

Comment:
>should append the hash here, just in case we have to change it. maybe tie into the other static bits in `index.js`?

We could add it at the point a change needs to be made, which speaks to a bigger problem of: if anyone were to need to make a change, would they realize they'd also need to update the cache flag? ...things we take for granted with the build process 😄